lib/Sema/SemaDecl.cpp

```diff
@@ -11394,47 +11496,47 @@ void Sema::ActOnUninitializedDecl(Decl *RealDecl) {
       else if (B && !B->isInvalid() && !B->isUnknown() && !Ty->isArrayType())
         Diag(Var->getLocation(), diag::err_initializer_expected_with_bounds)
           << Var;
-
-      // An unchecked pointer in a checked scope with a bounds expression must be initialized
-      if (Ty->isUncheckedPointerType() && InCheckedScope && Var->hasBoundsExpr())
-        Diag(Var->getLocation(), diag::err_initializer_expected_for_unchecked_pointer_in_checked_scope_with_bounds_expr)
-          << Var;
-
-      // An integer with a bounds expression must be initialized
-      if (Ty->isIntegerType() && Var->hasBoundsExpr())
-        Diag(Var->getLocation(), diag::err_initializer_expected_for_integer_with_bounds_expr)
-          << Var;
-
-      // struct/union and array with checked pointer members must have initializers 
-      // array with checked ptr element
-      if (Ty->isArrayType()) {
-        // if this is an array type, check the element type of the array, potentially with type qualifiers missing
-        if (Type::HasCheckedValue == Ty->getPointeeOrArrayElementType()->containsCheckedValue(InCheckedScope))
-          Diag(Var->getLocation(), diag::err_initializer_expected_for_array)
-          << Var;
-      }
-      // RecordType(struct/union) with checked pointer member
-      if (Ty->isRecordType()) {
-        const RecordType *RT = Ty->getAs<RecordType>();
-        switch (RT->containsCheckedValue(InCheckedScope)) {
-        default: 
-          break;  
-        case Type::HasCheckedValue: {
-          Diag(Var->getLocation(), diag::err_initializer_expected_for_record_with_checked_value)
-          << RT->getDecl()->getTagKind() << Var;
-          break;
-        }
-        case Type::HasIntWithBounds: {
-          Diag(Var->getLocation(), diag::err_initializer_expected_for_record_with_integer_member_with_bounds_expr)
-          << RT->getDecl()->getTagKind() << Var;
-          break;
-        }
-        case Type::HasUncheckedPointer: {
-          Diag(Var->getLocation(), diag::err_initializer_expected_for_record_with_unchecked_pointer_in_checked_scope)
-          << RT->getDecl()->getTagKind() << Var;       
-          break;
-        }       
-        }
+
```

Comment:
Did the white spacing on this get changed by your editor?    This code looks unchanged.  Please avoid unnecessary formatting changes, as it makes code harder to review.  It also makes merging the latest clang sources harder.